### DISCUSSION TITLE
Pending Database Migration?

### DISCUSSION THREAD

**Discussion by cmplant3210** [mastodon/mastodon#17056] [Q&A] [answered]
I recently upgraded my server to v3.4.4 but am now receiving the following in my dashboard. 
![1BDEBE1E-C325-4115-A2F6-95DF94FE0C97](https://user-images.githubusercontent.com/37702401/143586892-32ece280-3648-485f-88b0-9f6f263181f7.jpeg)

Is this an automated process and i just need to wait, or is this a manual process and what do i need to do to clear this off? Thanks in advance for the help!

**Comment by ClearlyClaire** [comment, marked as answer]
It is a manual process and you should be running `bundle exec rails db:migrate` as per the release notes.

**Reply by cmplant3210** [reply]
Thank you very much for the direction.  I see that step is included in the 3.4.2 instructions, however they are not on the 3.4.4 instructions.  I will make sure to look at previous releases for this instruction in the future.  
![Untitled](https://user-images.githubusercontent.com/37702401/143661184-4cce9f8a-a826-4a1e-96ae-af7052580b50.jpg)
